### ISSUE TITLE
Rats Vs Cats

### ISSUE THREAD

**Issue by ShimmeringHydra** [siggame/MegaMinerAI-Game-Ideas#67]
# The Game:
## Story
The rat kingdom is fighting a war and being pressured by the great ~~demon lords~~ cat lords! Heros have risen and compete and collaborate for honor and victory! This is the story fo your little rodent, making a name for themself in this medieval magical world. (or other, steampunk may be cool for example)

## To Win
You want to be the most honorable rat of them all! to win you have to be alive by the end of the game and have the most honor;
if honor is tied, the gold is considered.
if gold is tied, party size is considered, larger parties bing more influential and better.
if party size is tied, total value of gear is considered.
if all else fails, coin flip.

## The World
All rats start on the same tile, a city tile. Each player controls 1 rat and has multiple actions they can do at any given time. the map is from a further out perspective from past games like catastrophe, having a city exist within a tile. water tiles and different biomes may exist as well.

## Tiles and actions
On a city tile, one might recruit party members, work a cheap job / local quest, buy items, recover / stay the night at an inn, or other stuff etc.

One may equipt or swap out specific items for movement or defence purposes depending on the terrain, use items, use items on other rats to assist them and gain honor from their feats if they are in a tough spot, try to steal a kill on one of the cat lords, try to colonize a new region, etc.

all biome tiles are yet to be determined, but tundra, plains, wetlands, and cursed grounds may be a few for consideration.

## Map, Mechanics, and combat
Players will take their actions based on the prior world state and make their movements all at the same time. as a result monsters can be grosely overkilled, players may buy the same stock from a shop without prices changing, etc etc. Stemming from this, shops that sell items will increase the item price at that location for a time after that item is bought.

Players try to accumulate honor through kills and honorous deeds, determined later.

Actions covers many of the mechanics; you can assist other players or compete with them for a kill. the player who starts combat will have the aggro so following players can benefit from joining. kills will be determined by the player who does the most damage to an enemy.

there will be enemies besides the cat lords, but they aren't specified as of yet. EXP may be a thing to make parties less beneficial in the long run due to splitting exp, unless paying for higher level NPC's.

the map may be 30x30 tile size or larger. not yet determined and can vary a lot by which mechanics make it into the final product.

## Final Notes and TL;DR
The focus of the game is on having a single unit with complex interactions and detail in it, rather than a team fighting game of back and forth sudo-chess. A secondary focus was on a game idea with fair amounts of player interaction without direct combat like most prior games we have developed.

TL;DR
game is about rats wanting to be heros/famous and being sly and cunning against each other to do so.
